model/service.go

```diff
@@ -107,6 +112,23 @@ const (
 	ProtocolRedis Protocol = "Redis"
 )
 
+// AuthenticationPolicy defines authentication policy for port.
+type AuthenticationPolicy string
+
+const (
+	// AuthenticationEnable indicates authentication should be enable (on the
+	// associated port). This overrules mesh's AuthPolicy.
+	AuthenticationEnable AuthenticationPolicy = "enable"
```

Comment:
It may be worth having an 'API review' focused on consistency. 
The global AuthPolicy uses 'none' and mutualtls' - could we use the same here ? In future we may add other options 
(like 'auto' ), but we should be consistent in terms across istio.